index.js

```diff
@@ -3,5 +3,12 @@
  * @return {Boolean}
  */
 module.exports = function (filename) {
+  if (typeof filename !== 'string') {
+    throw new TypeError([
+      '`filename` should be an `String`,',
+      'but got `' + (typeof filename) + '`: ' + filename,
+    ].join(' '));
+  }
+
```

Comment:
other way will be to use neat-contract:

```js
var contract = require('neat-contract');

module.exports = function (filename) {
  contract('filename', String, filename); // it will throw TypeError with human-friendly message if filename is not string 
  return filename[0] === '.';
};

```